docs/stylesheets/extra.css

```
@@ -0,0 +1,769 @@
+  --md-footer-bg-color--dark: #070c12;
+  --md-footer-fg-color:       var(--sc-muted);
+}
+
+/* ----------------------------------------------------------------
+   Light scheme overrides
+   ---------------------------------------------------------------- */
+[data-md-color-scheme="default"] {
+  --md-primary-fg-color:        #065f46;
+  --md-primary-fg-color--light: #047857;
+  --md-primary-fg-color--dark:  #064e3b;
+  --md-primary-bg-color:        #ffffff;
```

Stylelint (17.9.0)

[error] Expected "#ffffff" to be "#fff" (color-hex-length)

(color-hex-length)